qt_helpers.py

```diff
@@ -127,48 +171,59 @@ def setMargin(self, x):
     QT_API = QT_API_PYSIDE
 
     deny_module('PyQt4')
+    deny_module('PyQt5')
 
-loaders = [_load_pyqt4, _load_pyside]
-if os.environ.get('QT_API') == QT_API_PYSIDE:
-    loaders = loaders[::-1]
-
-msgs = []
-
-# acutally do the loading
-for loader in loaders:
-    try:
-        loader()
-        # we set this env var, since IPython also looks for it
-        os.environ['QT_API'] = QT_API
-        QtCore = sys.modules[__name__ + '.QtCore']
-        QtGui = sys.modules[__name__ + '.QtGui']
-        break
-    except ImportError as e:
-        msgs.append(str(e))
-        pass
-else:
-    raise ImportError("Could not find a suitable QT installation."
-                      " Encountered the following errors: %s" %
-                      '\n'.join(msgs))
 
+QtCore = None
+QtGui = None
 
-def is_pyside():
-    return QT_API == QT_API_PYSIDE
 
+def reload_qt():
+
```

Comment:
Can you explain what this does in a docstring? I assume this is only meant to be used to reimport the same Qt binding (since trying to import several leads to trouble)